### PR TITLE
BAH-2681 | Add bahmni-mart deployment

### PR DESCRIPTION
**Pull request by amankrayush**
Bahmni-mart Deployment to environments.

## Jira
-----------------
https://bahmni.atlassian.net/browse/BAH-2681

### CODE REVIEW

File: values/demo.yaml [hunk 1]
```diff
@@ -157,3 +157,11 @@ abha-verification:
...
     labels:
       environment: demo
+
+bahmni-mart:
+  enabled: true
+  config:
+    CRON_TIME: "*/10 * * * *"
+    MART_DB_NAME: martdb
+  secrets:
+    MART_DB_HOST: "bahmni-dev-postgresql"
```
Comment (mradul1807) [inline values/demo.yaml]: DB name is wrong. It should have `demo` instead of `dev`

File: values/local.yaml [hunk 2]
```diff
@@ -101,3 +101,17 @@ implementer-interface:
...
   enabled: false
+
+bahmni-mart:
+  enabled: true
+  config:
+    CRON_TIME: "*/10 * * * *"
+    MART_DB_NAME: martdb
+    OPENMRS_DB_NAME: openmrs
+  secrets:
+    MART_DB_HOST: "bahmni-dev-postgresql"
```
Comment (mradul1807) [inline values/local.yaml]: It should be `bahmni-local-postgresql`

File: values/performance.yaml [hunk 3]
```diff
@@ -160,3 +160,11 @@ abha-verification:
...
     labels:
       environment: performance
+
+bahmni-mart:
+  enabled: true
+  config:
+    CRON_TIME: "*/10 * * * *"
+    MART_DB_NAME: martdb
+  secrets:
+    MART_DB_HOST: "bahmni-dev-postgresql"
```
Comment (mradul1807) [inline values/performance.yaml]: Incorrect DB name

File: values/qa.yaml [hunk 4]
```diff
@@ -157,3 +157,11 @@ abha-verification:
...
     labels:
       environment: qa
+
+bahmni-mart:
+  enabled: true
+  config:
+    CRON_TIME: "*/10 * * * *"
+    MART_DB_NAME: martdb
+  secrets:
+    MART_DB_HOST: "bahmni-dev-postgresql"
```
Comment (mradul1807) [inline values/qa.yaml]: Incorrect DB name

**Review by mradul1807 — APPROVED** [review]

File: values/demo.yaml (hunk 1 above)
Comment (amankrayush) [reply values/demo.yaml]: fixed now. Thanks !!

File: values/local.yaml (hunk 2 above)
Comment (amankrayush) [reply values/local.yaml]: fixed now. Thanks !!

File: values/performance.yaml (hunk 3 above)
Comment (amankrayush) [reply values/performance.yaml]: fixed now. Thanks !!

File: values/qa.yaml (hunk 4 above)
Comment (amankrayush) [reply values/qa.yaml]: fixed now. Thanks !!

**Review by mradul1807 — APPROVED** [review]

File: values/demo.yaml [hunk 5]
```diff
@@ -157,3 +157,11 @@ abha-verification:
   metadata:
     labels:
       environment: demo
+
+bahmni-mart:
+  enabled: false
+  config:
+    CRON_TIME: "*/10 * * * *"
```
Comment (pajmera4) [inline values/demo.yaml]: As discussed, keep cron time to 15 or 30 mins

**Review by pajmera4 — CHANGES_REQUESTED** [review]
Before merging, bahmni mart needs to be tested with clinical config

File: values/demo.yaml (hunk 5 above)
Comment (amankrayush) [reply values/demo.yaml]: we have kept cron time to 5 minutes as of now as discussed.

**Comment by amankrayush** [comment]
> Before merging, bahmni mart needs to be tested with clinical config

ok.

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**